ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "The runtime-audit-engine module: advanced usage"
+---
+
+{% raw %}
+
+## Enabling debugging logs
+
+### Falco
+
+By default, the log level is set to `debug`.
```

Comment:
According to the template:
```
        {{- if .Values.runtimeAuditEngine.debugLogging }}
        - name: DEBUG
          value: "true"
        {{- end }}
```

and config
```
properties:
  debugLogging:
    type: boolean
    description: |
      If true all outputs will print in stdout the payload they send.
```

`Debug` is not a default. This sentence is invalid

Reply:
I added `By default, the debug logging for `Falcosidekick` is disabled.` to the Falcosidekick section.